week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,32 +12,32 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
-
+  let isBigEnough = true;
```

Comment:
This will make the test pass but it won't actually check if the number is bigger than 10. 

Inside the `if` statement you have got to compare the `num` variable to the number `10` using the `===` operator. Give it a try!

Reply:
I tried this === and the test failed